demo/src/app/components/+datepicker/datepicker-section.list.ts

```diff
@@ -131,6 +132,14 @@ export const demoComponentContent: ContentSection[] = [
         component: require('!!raw-loader?lang=typescript!./demos/reactive-forms/reactive-forms.component.ts'),
         html: require('!!raw-loader?lang=markup!./demos/reactive-forms/reactive-forms.component.html'),
         outlet: DemoDatepickerReactiveFormsComponent
+      },
+      {
+        title: 'Outside click',
+        anchor: 'outside-click',
+        component: require('!!raw-loader?lang=typescript!./demos/outside-click/outside-click.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/outside-click/outside-click.html'),
+        description: `<p>You can close a datepicker out a field using outsideClick property</p>`,
```

Comment:
Datepicker closes after outside click by default. To change this behavior, use `outsideClick` property.